tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Utils/BearerTokenBuilder.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.IdentityModel.Tokens;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Utils
+{
+    public sealed class BearerTokenBuilder
+    {
+        private X509Certificate2 _signingCertificate;
+        private string _issuer = "www.google.com";
```

Comment:
I think the user of a real website address might lead to some confusion here. Could we just use the authority settings instead?

Reply:
we do use the authority settings when we call IssuedBy, but it needs a default value, maybe I should just default it to null... I'll default it to null

Reply:
👍